commcare_connect/opportunity/tasks.py

```diff
@@ -63,7 +65,31 @@ def add_connect_users(user_list: list[str], opportunity_id: str):
             username=user.username, defaults={"phone_number": user.phone_number, "name": user.name}
         )
         opportunity_access, _ = OpportunityAccess.objects.get_or_create(user=u, opportunity_id=opportunity_id)
-        invite_user(u, opportunity_access)
+        invite_user.delay(u, opportunity_access)
+
+
+@celery_app.task()
+def invite_user(user, opportunity_access):
```

Comment:
The user and opportunity_access params here needs to replaced with the pk's for user and access.

Reply:
thanks. is that due to json serialization or some other reason?

Reply:
Yeah, the json serialization would error out for these django objects.